go.mod

```diff
@@ -5,7 +5,9 @@ go 1.14
 require (
 	github.com/astarte-platform/astarte-go v0.90.2
 	github.com/cenkalti/backoff/v4 v4.1.1
+	github.com/cloudflare/cfssl v1.6.1
```

Comment:
wtf?

Reply:
Just a leftover :sweat_smile:, now it's been removed